src/homepageExperience/components/steps/cli/InitializeTokenClient.tsx

```diff
@@ -0,0 +1,137 @@
+// Libraries
+import React, {FC, useEffect, useMemo} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
+
+// Actions
+import {
+  createAuthorization,
+  getAllResources,
+} from 'src/authorizations/actions/thunks'
+
+// Selectors
+import {getOrg} from 'src/organizations/selectors'
+import {getMe} from 'src/me/selectors'
+import {getAllTokensResources} from 'src/resources/selectors'
+
+// Helper Components
+import CodeSnippet from 'src/shared/components/CodeSnippet'
+
+// Utils
+import {allAccessPermissions} from 'src/authorizations/utils/permissions'
+import {event} from 'src/cloud/utils/reporting'
+
+// Types
+import {AppState, Authorization} from 'src/types'
+
+// Tokens stuff
+
+type OwnProps = {
+  wizardEventName: string
+  setTokenValue: (tokenValue: string) => void
+  tokenValue: string
+}
+
+const collator = new Intl.Collator(navigator.language || 'en-US')
+
+// Client stuff
+
+export const InitializeTokenClient: FC<OwnProps> = ({
+  wizardEventName,
+  setTokenValue,
+  tokenValue,
+}) => {
+  const org = useSelector(getOrg)
+  const me = useSelector(getMe)
+  const allPermissionTypes = useSelector(getAllTokensResources)
+  const dispatch = useDispatch()
+  const url =
+    me.quartzMe?.clusterHost || 'https://us-west-2-1.aws.cloud2.influxdata.com/'
+  const orgID = org.id
+  const currentAuth = useSelector((state: AppState) => {
+    return state.resources.tokens.currentAuth.item
+  })
+  const token = currentAuth.token
+
+  const codeSnippet = `influx config create --config-name onboarding \n
+  --host-url "${url}" \n
+  --org "${orgID}" \n
```

Comment:
I think this is supposed to be org name and not orgID. 🤔

Reply:
@david-rusnak I think this should be `orgId`. Either will work, but I think going forward, when given the option of org name or org id, we should default to using org id, since there have been discussions about the difficulty of org name having to be unique.

Reply:
ah really? figma design shows org name. Also, I thought org name specifically was required to initialize client?

Reply:
i see both being used interchangeably. Never mind that ^